content/blog/2022/09/vimdap/index.md

```diff
@@ -0,0 +1,189 @@
+---
+title: "Camel Textual Route debugging in Vim"
+date: 2022-09-14
+authors: [Navyashree0923]
+categories: ["Tooling"]
+preview: "Camel Textual Route debugging in Vim"
+---
+
+A month before it became easier to provide language support inside Vim, Neovim using coc.nvim -[See this blog post](/blog/2022/07/coc-nvim-camel-lsp/). It is now time to use Camel Debug Adapter in Vim[camel-tooling/camel-dap-client-vim](https://github.com/camel-tooling/camel-dap-client-vim).
+
+# VIM Debug Adapter client for Apache Camel
+
+I just got the debug integration for camel in vim running using [vimspector](https://github.com/puremourning/vimspector).
+
+![Demo](images/vimdap.gif)
+
+## Install
+Install the vimspector using vim-plug plugin manager.
+- Inside `.vimrc` file:
```

Comment:
the rendering seems bad on the website. I think you need to add an empty line

![image](https://user-images.githubusercontent.com/1105127/190120224-d863fc3a-e0e8-46f0-afa4-cdb9e0b7eb7a.png)

Reply:
Thanks. I have updated the change